### DISCUSSION TITLE
reloading third party comment system | remark42

### DISCUSSION THREAD

**Discussion by uPagge** [squidfunk/mkdocs-material#5100] [Q&A]
Hello
I use Remark42 as my comment system. I noticed this is the problem, when I select another page through the god menu on the left, the iframe does not load on the new opened page. You have to reload the page, and then the comments appear. How can this be corrected?

https://remark42.com

**Comment by squidfunk** [comment]
Not sure. In general, it should be no problem and also work with instant loading. Please ask the maintainers of remark42.

**Reply by squidfunk** [reply]
As an additional note, if you need to do re-initialization of some sort when using third-party libraries and instant loading, please use:

``` js
document$.subscribe(() => {
  /* Triggered on initial page load and on each load after navigation when using instant loading */
})
```
